packages/cli-repl/src/mongosh-repl.spec.ts

```diff
@@ -352,6 +363,14 @@ describe('MongoshNodeRepl', function () {
     };
     const tabtab = async () => {
       await tab();
+      if (process.env.USE_NEW_AUTOCOMPLETE) {
+        // TODO: This is because autocomplete() is async and will either list
```

Comment:
Less of a TODO than a NOTE.

Reply:
Yeah, I feel like setting a fixed timeout of 210 is either almost guaranteed flakiness or something that at least looks like it would be flaky ... can we somehow wait for the right event (e.g. some autocompletion data fteched event) at this point?

Reply:
I updated the existing autocomplete event and added a new helper around that so we can wait until the event fires with results.